sw/device/silicon_owner/tock/apps/hello/src/hello.rs

```diff
@@ -0,0 +1,15 @@
+//! An extremely simple libtock-rs example. Just prints out a message
+//! using the Console capsule, then terminates.
+
+#![no_main]
+#![no_std]
+use core::fmt::Write;
+use libtock2::console::Console;
+use libtock2::runtime::{set_main, stack_size};
+
+set_main! {main}
```

Comment:
Shouldn't this be `set_main!(main);`? (Rust does not care what the delimiters for a macro call are.)